# ValueError 

# This occurs when a built-in operation or function recieves an argument that has the right type but an inappropriate value

int("foo")

# ValueError: invalid literal for int() with base 10: 'foo'